vagrant/provisioning/roles/haproxy/tasks/main.yml

```diff
@@ -194,7 +192,25 @@
         server arkcase0 {{ arkcase_host }}:8843 check fall 3 rise 2 check ssl verify required ca-file {{ ssl_ca }}
   
   register: arkcase_config
-  
+
+- name: FOIA fronted part
+  become: yes
+  lineinfile:
+    backup: yes      
+    path: "/etc/opt/rh/rh-haproxy18/haproxy/haproxy.cfg"
```

Comment:
Please verify what happens if you run this play multiple times. I'm pretty sure it will add these lines every time you run the play (run the play 5 times and you will have 5 acl_foia acl lines, and 5 use_backend be_foia lines).

If I'm right you can make this play idempotent by first detecting if the lines already exist.